be/src/exprs/runtime_filter_slots.h

```diff
@@ -220,7 +220,11 @@ class RuntimeFilterSlotsBase {
     Status publish() {
         for (auto& pair : _runtime_filters) {
             for (auto filter : pair.second) {
-                RETURN_IF_ERROR(filter->publish());
+                auto state = filter->publish();
```

Comment:
use status  not state

Reply:
This piece of code will be removed in the future, so fix the workflow as soon as possible